target_chains/cosmwasm/tools/src/wormhole-stub.ts

```diff
@@ -123,19 +108,36 @@ async function run() {
       },
     });
 
+    // 4 vaa upgrades for guardian set
+    Object.keys(VAA_UPGRADES).forEach((id) => {
```

Comment:
For mainnet only right?

Reply:
dev is doing the "testnets read mainnet feeds" thing we discussed for cosmwasm

Reply:
testnet will read from mainnet feeds